pkg/config/config.go

```diff
@@ -275,6 +278,24 @@ func (cfg *Config) checkUnsafeEndpoint(endpoint *url.URL) error {
 	return nil
 }
 
+// validateReconcileConfigResources validates the --reconcile-resource-resync-seconds flag
+// by checking the resource names and their corresponding duration.
+func (cfg *Config) validateReconcileConfigResources(valideResourceNames []string) error {
+	for _, resourceResyncSecondsFlag := range cfg.ReconcileResourceResyncSeconds {
+		resourceName, _, err := parseReconcileFlagArgument(resourceResyncSecondsFlag)
+		if err != nil {
+			return fmt.Errorf("error parsing flag argument '%v': %v. Expected format: resource=seconds", resourceResyncSecondsFlag, err)
+		}
+		if !ackutil.InStrings(resourceName, valideResourceNames) {
```

Comment:
FYI `lo.Contains` - https://github.com/samber/lo#contains. But I think we want to do case insensitive contains

Reply:
I'd like to do a special PR where we go and bring Contains everywhere. Probably after dealing with https://github.com/aws-controllers-k8s/pkg/pulls

Reply:
Works for me